addons/binding/org.openhab.binding.millheat/ESH-INF/thing/heater.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="millheat"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="heater">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+
+		<label>WIFI enabled heater</label>
+
+        <category>HVAC</category>
```

Comment:
https://www.eclipse.org/smarthome/documentation/concepts/categories.html

It should either be HVAC, RadiatorControl or none (since there isn't a perfect match IMHO)

Reply:
I did not find Heating in the link above - am I looking at the wrong place?

Reply:
Near the bottom, under "Control". You can omit category if you don't find a good fit.

Reply:
I think you are mixing Thing vs Channel categories. There is a channel category for Heating but not a thing category.
But I removed the category for the thing since HVAC isn't a perfect match.